### ISSUE TITLE
Cannot use 'npm start'

### ISSUE THREAD

**Issue by pchard** [electron/electron-quick-start#75]
Hi.
This has most likely been solved many times by you already, but I just can't get it to work.
```
> electron-quick-start@1.0.0 start /root/electron-quick-start
> electron .


npm ERR! Linux 4.4.0-31-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "start"
npm ERR! node v4.2.6
npm ERR! npm  v3.5.2
npm ERR! code ELIFECYCLE
npm ERR! electron-quick-start@1.0.0 start: `electron .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the electron-quick-start@1.0.0 start script 'electron .'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the electron-quick-start package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     electron .
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs electron-quick-start
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls electron-quick-start
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /root/electron-quick-start/npm-debug.log

```

**Comment by MarshallOfSound** [comment]
Have you run `npm install` ?

**Comment by pchard** [comment]
Yes.

**Comment by MarshallOfSound** [comment]
TLDR: Don't run on a VPS 👍
